storybook/.storybook/preview.js

```diff
@@ -60,3 +63,17 @@ export const parameters = {
     states: [],
   },
 };
+
+export const globalTypes = {
```

Comment:
It's great to know about this. Seems very straightforward and could be really useful in the future for implementing all sorts of custom controls (eg, I think Storybook has it's own responsiveness sizes that might not align with ours)